plugins/modules/lvg.py

```diff
@@ -179,6 +179,11 @@
     - Specifies the major number of the volume group that is created.
     - Can be used only when creating a volume group, hence when I(state=present).
     type: int
+  quorum:
+    description:
+    - Enables/disables quorum on the volume group.
+    - Can be used changing a volume group, hence when I(state=present).
```

Comment:
Use this statement instead: "Can be used while changing an existing volume group."

Reply:
This is resolved.